tcf_website/static/blog/blog.css

```
@@ -12,4 +17,8 @@
 .blog-body code {
   background-color: #f0f0f0;
   padding: 0.2em;
+}
+.blog-body blockquote {
+  border-left: 1px solid #6c757d;
+  padding: 1em;
```

```suggestion
  padding: 0 0.9em;
```
A litte less spacing on the top and bottom